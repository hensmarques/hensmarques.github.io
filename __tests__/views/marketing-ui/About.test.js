import React from 'react';
import About from '../../../src/views/marketing-ui/About';

describe('<About/>', () => {
  it('renders and matches the snapshot', () => {
    const wrapper = shallow(<About />);
    expect(wrapper).toMatchSnapshot();
  });
});
