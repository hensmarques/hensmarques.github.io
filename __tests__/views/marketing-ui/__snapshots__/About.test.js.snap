// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About/> renders and matches the snapshot 1`] = `
<div>
  <div
    id="about"
  >
    <Fade
      left={true}
    >
      <div>
        <img
          className="dotglobe"
          src="images/icons/dotglobe.svg"
        />
      </div>
    </Fade>
    <Fade
      right={true}
    >
      <div
        className="about-list-content"
      >
        <h5
          className=""
        >
          About Cointrader
        </h5>
        <p>
          Aenean lacinia bibendum nulla sed consectetur. Sed posuere consectetur est at lobortis. Maecenas faucibus mollis interdum. Maecenas sed diam eget risus varius blandit sit amet non magna. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. Vestibulum id ligula porta felis euismod semper. Donec id elit non mi porta gravida at eget metus. Aenean lacinia bibendum nulla sed consectetur. Cras justo odio, dapibus ac facilisis in, egestas eget quam.
        </p>
      </div>
    </Fade>
  </div>
  <div
    id="vision"
  >
    <Fade
      left={true}
    >
      <div
        className="about-list-content"
      >
        <h5
          className=""
        >
          About Us
        </h5>
        <p>
          Aenean lacinia bibendum nulla sed consectetur. Sed posuere consectetur est at lobortis. Maecenas faucibus mollis interdum. Maecenas sed diam eget risus varius blandit sit amet non magna. Vivamus sagittis lacus vel augue laoreet rutrum faucibus dolor auctor. Vestibulum id ligula porta felis euismod semper.
        </p>
        <h5
          className=""
        >
          Address
        </h5>
        <p>
          315 Madison Ave, 24th Floor
          <br />
          New York, NY 10017
        </p>
      </div>
    </Fade>
    <Fade
      right={true}
    >
      <div
        className=""
      >
        <img
          className="stackedsquares"
          src="images/icons/stackedsquares.svg"
        />
      </div>
    </Fade>
  </div>
</div>
`;
